newTestFile.py

```diff
@@ -0,0 +1,92 @@
+try:
+    import pyaudio
+    import numpy as np
+    import pylab
+    import matplotlib.pyplot as plt
+    import time
+    import sys
+except:

```

Comment:
Remove this catch. try catch should be used sparingly because silent errors are bad. Quite unusual to catch importas.